app/handlers/base.py

```diff
@@ -1,11 +1,16 @@
+__all__ = ['BaseHandler']
```

Comment:
This shouldn't be necessary as we never do `from baselayer.app.handlers.base import *`